cmake/windows/resources/installer-Windows.iss.in

```diff
@@ -62,3 +74,27 @@ begin
     Result := InstallPath
 end;
 
+
+// Remove old OBS-NDI conflicting plugin - by Trouffman for DistroA https://github.com/DistroAV/DistroAV/
+
+// CAREFUL : this target the old AppID.
+const UninstallRegisterPath = 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{A9039243-4FE7-45E7-8B11-7DC1ACB67B9D}_is1';
+
+function GetOldAppUninstallerPath(Param: String): String;
+var
+  UninstallerPathRegistry: String;
+// Param is required to be used in other sections.
+begin
+  result := '';
+  // Check primary registry location
+  if ( RegQueryStringValue(HKLM, UninstallRegisterPath, 'UninstallString', UninstallerPathRegistry) ) then
+      result := RemoveQuotes(UninstallerPathRegistry)
+  // Check alternative Registry location
+  else if ( RegQueryStringValue(HKCU, UninstallRegisterPath, 'UninstallString', UninstallerPathRegistry) ) then
+      result := RemoveQuotes(UninstallerPathRegistry);
+end;
+
+function UninstallOldAppAvailable(): Boolean;
+begin
+  if (GetOldAppUninstallerPath('') <> '') then Result := True else Result := False;
```

Comment:
it has been a LOOOOOONG time since I programmed Pascal!
Does case matter?
You use `result` in some places and `Result` in others.
Make them consistent.

Reply:
Does not seems to matter in my tests.

Will adjust to have all the same lowercase.

Reply:
I looked into the details and all the examples. use the `Results` with capitalised R. this has been unified in new commit.